build/hero_abilities.json

```diff
@@ -0,0 +1,4855 @@
+{
+  "npc_dota_hero_antimage": {
+    "abilities": [
+      "antimage_mana_break",
+      "antimage_blink",
+      "antimage_spell_shield",
+      "antimage_mana_void"
+    ],
+    "talents": [
+      {
```

Comment:
saves space, but is less explicit. Assumes user knows, or will guess, they are ordered that way.
I'm 50/50, I'd rather you decide :D 
Let me know if you want me to resubmit with just list of strings